ledfx/utils.py

```diff
@@ -1742,7 +1742,7 @@ def is_snake_case(string) -> bool:
     Returns:
         bool: True if the string is in snake_case format, False otherwise.
     """
-    return re.match("^[a-z][a-z_]*[a-z]$", string) is not None
+    return re.match("^[a-z][a-z0-9_]*[a-z0-9]$", string) is not None
```

Comment:
I don't think keys should ever contain digits - what is a use case for these?

Reply:
snake case allows numbers to my understanding, so I just considered it an oversight.

The specific use case that tripped me up is where in the text development I have some VERY generic switches or sliders that do different things in the different text effects, so I have for example bool 

option_1

and 

option_2

Reply:
@shauneccles I don't want to just shove this in, do you understand what I am stating, in that if its "snake case" it should be "snake case"

Reply:
Soz I been busy - I'm just not sure when a key should ever have a digit in it as opposed to spelt out - option_1 or option_2 are just terrible keys.

Key should outline what they do/are, and I'm not sure if there's every a case where that involves a digit?